src/store/storage-controller.ts

```diff
@@ -111,19 +109,46 @@ export class StorageController {
     return messages;
   }
 
-  public static async delete(
-    messageStore: MessageStore,
-    dataStore: DataStore,
+  public async delete(
     tenant: string,
     message: BaseMessage
   ): Promise<void> {
     const messageCid = await Message.getCid(message);
 
     if (message.descriptor.dataCid !== undefined) {
-      await dataStore.delete(tenant, messageCid, message.descriptor.dataCid);
+      await this.dataStore.delete(tenant, messageCid, message.descriptor.dataCid);
     }
 
-    await messageStore.delete(tenant, messageCid);
+    await this.messageStore.delete(tenant, messageCid);
+  }
+
+  public getMessage(tenant: string, messageCid: string): Promise<BaseMessage|undefined> {
+    return this.messageStore.get(tenant, messageCid);
+  }
+
+  public putMessage(
+    tenant: string,
+    messageJson: BaseMessage,
+    indexes: Record<string, string>,
+    options?: MessageStoreOptions
+  ): Promise<void> {
+    return this.messageStore.put(tenant, messageJson, indexes, options);
+  }
+
+  public queryMessages(tenant: string, filter: Filter, options?: MessageStoreOptions): Promise<BaseMessage[]> {
+    return this.messageStore.query(tenant, filter, options);
+  }
+
+  public getData(tenant: string, messageCid: string, dataCid: string): Promise<GetResult|undefined> {
+    return this.dataStore.get(tenant, messageCid, dataCid);
+  }
+
+  public appendEvent(tenant: string, messageCid: string): Promise<string>|undefined {
+    return this.eventLog.append(tenant, messageCid);
+  }
+
+  public deleteEvents(tenant: string, deletedMessageCids: string[]): Promise<number>|undefined {
```

Comment:
ditto ([:146](https://github.com/TBD54566975/dwn-sdk-js/pull/338/files#diff-cbe02e4d9564ce57729575f2fce1680efec7c8f72fe777276201b785fc0b3e8dR146))
```suggestion
  public deleteEvents(tenant: string, deletedMessageCids: string[]): Promise<number> {
```

Reply:
Obsoleted by https://github.com/TBD54566975/dwn-sdk-js/pull/338/commits/f872016b3691c4b2f4465ca59aaa33507847a820